libbeat/tests/system/beat/beat.py

```diff
@@ -29,6 +31,17 @@ def start(self):
             stderr=subprocess.STDOUT)
         return self.proc
 
+    def kill(self):
+        if sys.platform.startswith("win"):
```

Comment:
Adding this for all beats will address #599, but you also need to modify the start method for Windows. See https://github.com/elastic/beats/commit/ee446a4036aa97b96147d45225655dfb850fef3b#diff-714b8651f2c293a867918ab7def84ec0L25